Dockerfile

```diff
@@ -1,20 +1,46 @@
-# docker build -t amphitheatre-playground:v0.0.1 .
-FROM node:18-buster-slim AS build
+#
+# docker build -t ghcr.io/amphitheatre-app/playground:v0.0.1 .
+#
+FROM node:21-slim AS base
 
-WORKDIR /app/playground
-COPY package*.json ./
-RUN npm i
+# Install dependencies only when needed
+FROM base AS deps
+WORKDIR /home/app
```

Comment:
How about we look at other mainstream practices and standardize on `WORKDIR /app`?